exporter/terra/dcp_staging_client.py

```diff
@@ -61,10 +61,14 @@ def __init__(self, gcs_storage: GcsStorage, gcs_xfer: GcsXferStorage, schema_ser
         self.schema_service = schema_service
         self.ingest_client = ingest_client
 
-    def transfer_data_files(self, submission: Dict, project_uuid, export_job_id: str):
+    def transfer_data_files(self, submission: Dict, project_uuid, export_job_id: str) -> (TransferJobSpec, bool):
         upload_area = submission["stagingDetails"]["stagingAreaLocation"]["value"]
         bucket_and_key = self.bucket_and_key_for_upload_area(upload_area)
```

Comment:
instead of populateing an array and derefrencing by index, you could populate 2 variables:

```python
bucket, key = ...
```